packages/meditrak-app/package.json

```diff
@@ -109,7 +109,7 @@
     "react-navigation": "^1.5.8",
     "react-navigation-redux-helpers": "^1.0.3",
     "react-redux": "^5.0.7",
-    "realm": "^2.29.1",
+    "realm": "^6.1.0",
```

Comment:
Did you look at release notes for realm to check if there were breaking changes we need to be aware of? Most of their changes over the last years have been related to their internal sync module, which we don't use, but it'd be worth looking to see if there's anything we do need to change.

Reply:
Great point, I will have a look! I haven't been able to try the app locally yet because of some issues in my environment, but will also do